src/main/java/org/synyx/urlaubsverwaltung/statistics/vacationoverview/web/ApplicationForLeaveVacationOverviewViewController.java

```diff
@@ -27,15 +51,23 @@
 @Controller
 public class ApplicationForLeaveVacationOverviewViewController {
```

Comment:
Für diese Klase fehlen noch testcases - nachziehen? Oder ggfs in kleinere Methoden packen?

Reply:
sicknotes sollten jetzt abgedeckt sein.

fehlt noch:
* bestätigter urlaub
* noch nicht bestätigter urlaub
* wochenende
* feiertage (falls die im kalender hervorgehoben werden)